src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BlobProcessorTest.java

```diff
@@ -36,7 +36,7 @@ public void should_process_zipfile_after_upload_and_set_status() {
     }
 
     @Test
-    public void should_process_zipfile_with_supplementary_evidence_with_oce_classification() {
+    public void should_process_zipfile_with_supplementary_evidence_with_ocr_classification() {
```

Comment:
not related to the change, but noticed when adding the rescan_for property to the existing functional test data